pkg/cases/handler.go

```diff
@@ -8,12 +8,39 @@ import (
 type CaseHandler struct {
 }
 
+// @Summary Get the lastest case ID
+// @Description
+// @Tags Case
+// @Produce json
+// @Param patientID path uint true "patient ID"
+// @Success 200 {integer} api.ReturnedData{data=uint}
+// @Router /{patientid}/case [GET]
+func (h *CaseHandler) GetLastCaseID(c echo.Context) error {
+	// ...
+	c.Logger().Debug("hello world")
+	return c.JSON(200, api.Return("GetLastCaseID", nil))
+}
+
+// @Summary Get a case ID list
+// @Description
+// @Tags Case
+// @Produce json
+// @Param patientID path uint true "patient ID"
+// @Param department path string false "department name" nil
+// @Success 200 {array} api.ReturnedData{data=[]uint}
+// @Router /{patientid}/case/department [GET]
```

Comment:
Not a good routing too. And btw, I don't quite get why a `department` is necessary here.
And the same thing, you shouldn't just return an ID. An ID is useless for the frontend, you have to make another request to get the data, which is a waste.